index.html

```diff
@@ -574,6 +574,24 @@ <h3>Cheat Sheet:</h3>
           >
         </p>
       </div>
+      <div class="emoji">
+        <div id="akankshach"></div>
+        <p>
+          by
+          <a href="https://twitter.com/ch_akanksha" target="_blank"
+            >@ch_akanksha</a
+          >
+        </p>
+      </div>
+      <div class="emoji">
+        <div id="Jomar2019">&#129313;</div>
```

Comment:
your emoji should not have a child. the emoji should be drawn using css :)

Reply:
okay thanks